backend/Dockerfile

```diff
@@ -13,10 +16,9 @@ WORKDIR /backend
 # Copy only dependencies
 COPY pyproject.toml poetry.lock* ./
 
-# Install dependencies
-# --only main: Skip installing packages listed in the [tool.poetry.dev-dependencies] section
+# Install all dependencies
 RUN poetry config virtualenvs.create false \
-    && poetry install --no-interaction --no-ansi --only main
```

Comment:
why is this changing?

Reply:
So that pytest is available for those who develop inside a docker container and want to run tests inside.